back/src/v1/capacities.ts

```diff
@@ -0,0 +1,16 @@
+import useDatabase from '@composables/useDatabase';
+import useUtils from '@composables/useUtils';
+import express from 'express';
+
+const { database } = useDatabase();
+const { fromDbFormat, sendResponse } = useUtils();
+
+const router = express.Router();
+
+router.get('/', async (_, res) => {
+  const data = await database.capacity.findMany();
+
+  sendResponse(res, fromDbFormat(data), 200);
```

Comment:
If the database suddenly shut down, which error do you return?

Reply:
You're right i need to try catch this one